.circleci/config.yml

```diff
@@ -372,13 +372,9 @@ commands:
   pip_install:
     description: "Install NuCypher with Pip"
     steps:
-      - run:
-          name: Install Python Dependencies with Pip
-          command: pip3 install --user -e . -r requirements.txt
-      - check_nucypher_entrypoints   # Ensure Standard Installation Entry-points Work
```

Comment:
What about checking the entry points?

Reply:
Let me revisit this and make an update.

Reply:
Okay, I took another look and I think you are misreading the diff (Unless I am missing something)... we are still checking entry points a few lines below this one in the same job.  were we calling this check twice unintentionally?  ( also @derekpierre )

Reply:
Ah I didn't see that. All good then 👍

Reply:
👍 - I missed it as well